src/hotkeys_provider.rs

```diff
@@ -115,6 +99,34 @@ pub fn HotkeysProvider(
         disable_scope,
         toggle_scope,
     });
+}
+
+pub fn use_hotkeys_context() -> HotkeysContext {
+    use_context::<HotkeysContext>().expect("expected hotkeys context")
+}
+
+#[allow(unused_variables)]
+#[component]
+pub fn HotkeysProvider(
```

Comment:
Makes sense to me 👍🏼 

In fact, the `HotkeysProvider` could be removed entirely if `provide_hotkeys_context` could accept a element ref, so the root `<div>` tag wouldn't be needed, thus preventing any side effects that DOM element could introduce. But I'm sure that this would be a change that could be introduced later.

Reply:
Yes, removing the root `<div />` was something I had in mind for a long time, but also wanted a intuitive set up, hence the `<HotkeysProvider />`. 

However, this might be much better